go.mod

```diff
@@ -333,5 +331,6 @@ replace (
 
 	// Using private fork of controller-tools. See commit msg for more context
 	// as to why we are using a private fork.
-	sigs.k8s.io/controller-tools => github.com/cilium/controller-tools v0.16.1-1
+	// For testing purpose, use another fork to include https://github.com/kubernetes-sigs/controller-tools#1061
+	sigs.k8s.io/controller-tools => github.com/sayboras/controller-tools v0.0.0-20240925150353-20695830368c
```

Comment:
wait for a sync to happen in cilium/controller-tools to pick up https://github.com/kubernetes-sigs/controller-tools/pull/1061